blocks/small-manual-promo-block/features/small-manual-promo/default.jsx

```diff
@@ -1,105 +1,53 @@
-
-const HeadlineText = styled.h2`
-  font-family: ${(props) => props.primaryFont};
-`;
-
-const SmallManualPromoItem = ({ customFields = {} }) => {
-  const { showImage = true } = customFields;
-  const { arcSite, isAdmin } = useFusionContext();
+const SmallManualPromoItem = ({ customFields }) => {
   const { searchableField } = useEditableContent();
-  const imagePosition = customFields?.imagePosition || 'right';
-
-  const headlineMarginClass = getPromoStyle(imagePosition, 'headlineMargin');
-
-  const imageMarginClass = getPromoStyle(imagePosition, 'margin');
-  const resizedImageOptions = useContent({
-    source: (customFields.lazyLoad || isAdmin) ? 'resize-image-api-client' : 'resize-image-api',
-    query: { raw_image_url: customFields.imageURL, 'arc-site': arcSite },
-  });
-  const ratios = ratiosFor('SM', customFields.imageRatio);
-
-  const promoContainersStyles = {
-    containerClass: getPromoStyle(imagePosition, 'container'),
-    headlineClass: showImage
-      ? 'col-sm-xl-8'
-      : 'col-sm-xl-12',
-    imageClass: 'col-sm-xl-4',
-  };
+  const { isAdmin } = useFusionContext();
 
-  const renderWithLink = useCallback((element, props, attributes) => (
-    <a
-      href={customFields.linkURL || '#'}
-      className={(props && props.className) || ''}
-      target={customFields.newTab ? '_blank' : '_self'}
-      rel={customFields.newTab ? 'noreferrer noopener' : ''}
-      onClick={!customFields.linkURL ? (evt) => {
-        evt.preventDefault();
-      } : undefined}
-      {...attributes}
-    >
-      {element}
-    </a>
-  ), [customFields.linkURL, customFields.newTab]);
+  const imagePosition = customFields?.imagePosition || 'right';
+  const headlineMarginClass = SmallPromoStyles(imagePosition, 'headlineMargin');
 
-  const headline = customFields.showHeadline && customFields.headline
+  const headlineOutput = customFields.showHeadline && customFields.headline
     && (
-      <div className={`promo-headline ${headlineMarginClass}`}>
-        { renderWithLink((
-          <HeadlineText
-            primaryFont={getThemeStyle(arcSite)['primary-font-family']}
-            className="sm-promo-headline"
-          >
-            {customFields.headline}
-          </HeadlineText>
-        ), { className: 'sm-promo-headline' }) }
-      </div>
+      <PromoHeadline
+        text={customFields.headline}
+        link={customFields.linkURL}
+        className={headlineMarginClass}
+        linkClassName="sm-promo-headline"
+        headingClassName="sm-promo-headline"
+        newTab={customFields.newTab}
+      />
     );
 
-  const image = showImage && customFields.imageURL && resizedImageOptions
+  const image = customFields?.showImage && customFields.imageURL
     && (
-      <div
-        className={imageMarginClass}
-        {...searchableField('imageURL')}
-      >
-        { renderWithLink(
-          <Image
-            url={customFields.imageURL}
+      <div style={{ position: isAdmin ? 'relative' : null }}>
```

Comment:
isAdmin affecting position? not sure I follow here

Reply:
This is need for integrated photo search - this is as per before - just had to move it's location with the refactor, it's for the "replace image" button to work in the current editor experience